include/z64cutscene.h

```diff
@@ -344,6 +344,42 @@ typedef enum CutsceneDestination {
     /* 0x77 */ CS_DEST_ZELDAS_COURTYARD_RECEIVE_LETTER
 } CutsceneDestination;
 
+/*
+correlation with `enum SceneLayer`:
+  CS_INDEX_MANUAL:     [SCENE_LAYER_CHILD_DAY .. SCENE_LAYER_ADULT_NIGHT]
+  CS_INDEX_SCRIPTED_*: SCENE_LAYER_CUTSCENE_FIRST
+
+correlation with `enum EntranceIndex`
+  CS_INDEX_MANUAL:     base entrance index + scene layer
+  CS_INDEX_SCRIPTED_*: base entrance index + SCENE_LAYER_CUTSCENE_FIRST + (cutscene index & 0xF)
```

Comment:
```suggestion
correlation with `enum SceneLayer`:
  CS_INDEX_MANUAL:     [SCENE_LAYER_CHILD_DAY .. SCENE_LAYER_ADULT_NIGHT]
  CS_INDEX_SCRIPTED_*: SCENE_LAYER_CUTSCENE_FIRST + (cutscene index & 0xF)

correlation with `enum EntranceIndex`
  CS_INDEX_MANUAL:     base entrance index + scene layer
  CS_INDEX_SCRIPTED_*: base entrance index + scene layer
```

The comments here are a bit strange. sceneLayer is computed first, and the `CS_INDEX_SCRIPTED_*` condition should be `SCENE_LAYER_CUTSCENE_FIRST + (cutscene index & 0xF)`

After sceneLayer is calculated, cutsceneIndex has no effect on the final entrance index calculation.